### PR TITLE
[WM-2183] Remove CBAS-UI references

### PR DESCRIPTION
**Pull request by salonishah11**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/WM-2183

### What

- Removes CBAS UI references from Cromwell app on Azure

### Why

- Now that CBAS UI is deprecated we no longer need to deploy cbas-ui pod in Cromwell on Azure app

## Testing these changes
Tested these and helm chart changes (PR https://github.com/broadinstitute/cromwhelm/pull/276) in a BEE.

Cromwell on Azure doesn't have cbas-ui deployment
```
cromwhelm % kubectl get pods -n x7wagq-coa-ns                                 
NAME                                                      READY   STATUS    RESTARTS      AGE
coa-x7wagq-coa-rls-cbas-depl-5667885ddf-jksjj             1/1     Running   0             21h
coa-x7wagq-coa-rls-cromwell-depl-59d94d9bc6-djfqx         1/1     Running   3 (21h ago)   21h
coa-x7wagq-coa-rls-postgres-depl-795cdb9f7-kn7f2          1/1     Running   0             21h
coa-x7wagq-coa-rls-reverse-proxy-deploy-cbb56595d-lbkff   1/1     Running   0             78m
coa-x7wagq-coa-rls-tes-depl-ffb9df765-lwbnp               1/1     Running   2 (21h ago)   21h
listener-68dc84c7d8-9vgd8                                 1/1     Running   0             21h
```

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

**Comment by salonishah11** [comment]
jenkins retest

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala [hunk 1]
```diff
@@ -14,15 +14,14 @@ import org.broadinstitute.dsde.workbench.leonardo.config.Config.{appMonitorConfi
...
-  CbasUiDAO,
   CromwellDAO,
   HailBatchDAO,
   SamDAO,
   WdsDAO,
   WsmApiClientProvider,
   WsmDao
 }
-import org.broadinstitute.dsde.workbench.leonardo.db.{DbReference, KubernetesServiceDbQueries, SaveKubernetesCluster, _}
+import org.broadinstitute.dsde.workbench.leonardo.db._
```
Comment (cjllanwarne) [inline http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala:24]: This seems unrelated to removing CBAS UI?

**Review by cjllanwarne — APPROVED** [review]

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala (hunk 1 above)
Comment (salonishah11) [reply http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala:24]: Yes this was updated when I had to run `scalafmt`

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitorSpec.scala [hunk 2]
```diff
@@ -234,7 +232,7 @@ class LeoMetricsMonitorSpec extends AnyFlatSpec with LeonardoTestSuite with Test
         .unsafeRunSync()(IORuntime.global)
     // An up and a down metric for 7 services: 2 cbases, cbas-ui, cromwell, cromwell-reader, cromwell-runner, galaxy
     test.size shouldBe 14
```
Comment (dillydally414) [inline http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitorSpec.scala]: Weird that the size of this is still 14... I would think it should be 12 without cbas-ui

Comment (salonishah11) [reply http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitorSpec.scala]: Hmm good point. I will take a look at this.

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala (hunk 1 above)
Comment (rtitle) [reply http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala:24]: Btw I removed this entire file in my other PR. It was useful for testing aks apps early on, but I don’t think it works anymore.

**Review by rtitle — APPROVED** [review]

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitorSpec.scala (hunk 2 above)
Comment (salonishah11) [reply http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitorSpec.scala]: Thank you for pointing this out. I had missed removing a cbas-ui reference from the test file.

**Comment by salonishah11** [comment]
jenkins retest
